test/node/conversation-items.js

```diff
@@ -135,7 +132,31 @@ describe('Conversation Items', () => {
         assert(!res.text.likedByUsers || !res.text.likedByUsers.includes(user.userId));
     });
 
-    it('should unlike item and raise an itemUpdated event', async () => {
+    it('should mark items as read', async () => {
```

Comment:
You should also check the event that indicates items are marked as read (as long as there were unread items before)

Reply:
done